kubejs/assets/minecraft/lang/en_us.json

```diff
@@ -29,6 +33,8 @@
     "block.minecraft.gold_block": "Block of Yellow",
     "block.minecraft.iron_block": "Block of White",
     "block.minecraft.diamond_ore": "Light Blue Ore",
+    "item.minecraft.redstone_dust": "Red Dust",
```

Comment:
I do want redstone to stay redstone, it has a personality id like to keep

Reply:
changed!